lib/sepa.test.js

```diff
@@ -30,3 +36,146 @@ describe('IBAN tests',
       expect(SEPA.validateIBAN('DE1Zsantander')).toBe(false);
     });
   });
+
+describe('xml generation for transfer documents', () => {
+  const PAIN_FOR_TRANSFERS = 'pain.001.003.03';
+
+  function validTransferDocument(overrides = {}) {
```

Comment:
Hello @kewisch , I am not a Javascript developer, so I do not know if this is an idiomatic approach to provide optional overrides. If this is not so idiomatic, can you help me and show me how one would do this in js?

Reply:
If it is just a few parameters you can simply use parameters with defaults.

`function validTransferDocument(debtorId =A_VALID_CREDITOR_ID, debtorName="debtor-name")`

Otherwise using an object is fine. You can destructure it using 

`function validTransferDocument({ debtorId, debtorName }) {`

And even assign defaults right there:

`function validTransferDocument({ debtorId=A_VALID_CREDITOR_ID, debtorName='debtor-name' }) {`

This allows you to avoid the if() later on.

Reply:
Note for the record, JavaScript also [has an operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment) that allows you to assign a default value, but you won't need it if you use the default parameters.